### PR TITLE
Library Image info for website & mobile endpoints!

### PR DESCRIPTION
**Pull request by ahsu1230**
If a library has an image,
`FullLibraryInfo` will return the image information (pathUrl, x, y)

it should look like this:

```
{ "library" : 
  {
    ...
    "image" : { "path" : "library/asdfasdfasdf_600x400.jpg", "x" : 50, "y" : 50}
    ...
  }

}
```

`FullLibraryInfo` only comes from 4 endpoints, where imageSize is optional :

```
GET     /site/users/:userStr/libraries/:slug?is=wxh
GET     /site/libraries/:id?is=wxh
GET     /m/1/users/:userStr/libraries/:slug?is=wxh
GET     /m/1/libraries/:id?is=wxh
```

@andrewconner @2is10 @jaredpetker @thassss @joshm1

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala [hunk 1]
```diff
@@ -200,7 +201,7 @@ class LibraryCommander @Inject() (
     }
   }
 
-  def createFullLibraryInfos(viewerUserIdOpt: Option[Id[User]], showPublishedLibraries: Boolean, maxMembersShown: Int, maxKeepsShown: Int, idealKeepImageSize: ImageSize, libraries: Seq[Library]): Future[Seq[FullLibraryInfo]] = {
+  def createFullLibraryInfos(viewerUserIdOpt: Option[Id[User]], showPublishedLibraries: Boolean, maxMembersShown: Int, maxKeepsShown: Int, idealKeepImageSize: ImageSize, libraries: Seq[Library], idealLibraryImageSize: ImageSize): Future[Seq[FullLibraryInfo]] = {
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala:204]: I was thinking of having `idealLibraryImageSize` be optional. But then it just seems weird to me that `idealKeepImageSize` is not optional as well =/
Any thoughts for API?

Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala:204]: On second thought, maybe make a default value for `idealLibraryImageSize`

**Comment by joshm1** [comment]
lgtm

**Comment by 2is10** [comment]
:+1:

**Comment by andrewconner** [comment]
Great!

**Comment by ahsu1230** [comment]
sorry @2is10 @joshm1 still running tests. Sbt is being very slow =[
Once I confirm they're good, I'll merge. Thanks for looking it over!

**Pull request merged**